### PR TITLE
Remove str_eq

### PR DESCRIPTION
**Pull request by benjaminrsherman**

### CODE REVIEW

File: src/deroff.rs [hunk 1]
```diff
@@ -840,6 +840,22 @@ impl Deroffer {
             None => panic!("do_line` called with empty string as argument"),
         }
     }
+
+    fn str_eq(&self, offset: usize, other: &str, len: usize) -> bool {
```
Comment (scooter-dangle) [inline src/deroff.rs]: Can remove this guy now. :disappointed: womp womp

Comment (benjaminrsherman) [reply src/deroff.rs]: Removed.

Comment (pickfire) [reply src/deroff.rs]: This guy is not used in other functions.

**Comment by pickfire** [comment]
@benjaminrsherman Thanks for sending in the patch but the function is not used anymore, so let's close this.

**Pull request closed without merging**